.travis.yml

```diff
@@ -14,13 +14,18 @@ env:
   matrix:
     - GROUP=js
     - GROUP=js_cov
-    - GROUP=py3
-    - GROUP=py2
+    - GROUP=python PYTHON=2
+    - GROUP=python PYTHON=3.3
+    - GROUP=python PYTHON=3.4
+    - GROUP=python PYTHON=3.5
+    - GROUP=python PYTHON=3
```

Comment:
Will this explode our CI even more?

Reply:
Yes, unfortunately. Do you think it's worth it to test it?

Reply:
It adds three more runs of the python group, and one more run of the cli group.

Reply:
At the very least we can get rid of 3.3. IPython dropped support a week ago: https://github.com/ipython/ipython/pull/10833

Reply:
It's easy enough to ratchet this back if it gets unreasonable.

Reply:
Looks like 3.3 and 3.4 will have problems using the conda-forge packages. Here's the install error with 3.3:

```
UnsatisfiableError: The following specifications were found to be in conflict:
  - notebook -> ipykernel -> ipython[version='>=4.0'] -> pickleshare -> python=3.5 -> tk=8.6
  - notebook -> ipykernel -> ipython[version='>=4.0'] -> pickleshare -> python=3.5 -> xz[version='>=5.2.3,<6.0a0']
  - python=3.3
```
I'll remove 3.3 and 3.4 from the testing. That gives us the last two minor versions of 3 for testing.